### PR TITLE
Update localization.br.lua

### PR DESCRIPTION
**Pull request by anon1231823**

### CODE REVIEW

File: DBM-GUI/localization.br.lua [hunk 1]
```diff
@@ -220,91 +404,282 @@ L.DontShowFarWarnings		= "Não exibir anúncios/temporizadores para eventos que
...
 L.EnterProfileName			= "Entre com o nome do perfil"
 L.CreateProfile				= "Criar perfil para opções do núcleo do BDM"
 L.Area_ApplyProfile			= "Definir o perfil ativo para o núcleo do DBM"
 L.SelectProfileToApply		= "Selecionar o perfil a aplicar"
-L.Area_CopyProfile			= "Copiar o perfil de opções do núcleo do DBM"
-L.SelectProfileToCopy		= "Escolha o perfil à ser copiado"
-L.Area_DeleteProfile		= "Remover o perfil de opções do núcleo do DBM"
-L.SelectProfileToDelete		= "Escolha o perfil à ser removido"
+L.Area_CopyProfile			= "Copiar o perfil para o núcleo do DBM"
+L.SelectProfileToCopy		= "Selecionar o perfil à ser copiado"
```
Comment (Zidras) [inline DBM-GUI/localization.br.lua]:
```suggestion
L.SelectProfileToCopy		= "Selecionar o perfil a ser copiado"
```

File: DBM-GUI/localization.br.lua [hunk 2]
```diff
@@ -220,91 +404,282 @@ L.DontShowFarWarnings		= "Não exibir anúncios/temporizadores para eventos que
...
+L.Area_DeleteProfile		= "Remover o perfil para o núcleo do DBM"
+L.SelectProfileToDelete		= "Selecionar o perfil à ser removido"
```
Comment (Zidras) [inline DBM-GUI/localization.br.lua]:
```suggestion
L.SelectProfileToDelete		= "Selecionar o perfil a ser removido"
```

File: DBM-GUI/localization.br.lua [hunk 3]
```diff
@@ -39,13 +39,13 @@ L.MoveMe 					= "Mova-me"
 L.Button_OK 				= "OK"
 L.Button_Cancel 			= "Cancelar"
 L.Button_LoadMod 			= "Carregar AddOn"
-L.Mod_Enabled				= "Habilitar: %s"
+L.Mod_Enabled				= "Ativar: %s"
 L.Mod_Reset					= "Carregar opções padrão"
-L.Reset 					= "Resetar"
```
Comment (Zidras) [inline DBM-GUI/localization.br.lua:44]: I would keep it Resetar. It is an anglicism, sure, but one that most brazillians I've played with use.

File: DBM-GUI/localization.br.lua [hunk 4]
```diff
@@ -39,13 +39,13 @@ L.MoveMe 					= "Mova-me"
...
+L.Reset 					= "Redefinir"
 L.Import					= "Importar"
 
-L.Enable					= "Habilitar"
-L.Disable					= "Desabilitar"
+L.Enable					= ENABLE
```
Comment (Zidras) [inline DBM-GUI/localization.br.lua]: Since it's a global, doesn't need to be translated

File: DBM-GUI/localization.br.lua [hunk 5]
```diff
@@ -39,13 +39,13 @@ L.MoveMe 					= "Mova-me"
...
+L.Disable					= DISABLE
```
Comment (Zidras) [inline DBM-GUI/localization.br.lua]: Since it's a global, doesn't need to be translated

File: DBM-GUI/localization.br.lua [hunk 6]
```diff
@@ -66,67 +66,123 @@ L.Area_ProfilesSetup		= "Guia de uso de perfis DBM"
...
+L.Latency_Text				= "Latência máx. de sincronização: %d"
 
-L.Button_RangeFrame			= "Exibir/esconder quadro de distância"
-L.Button_InfoFrame			= "Exibir/esconder quadro de informações"
+L.Button_RangeFrame			= "Exibir/ocultar quadro de distância"
+L.Button_InfoFrame			= "Exibir/ocultar quadro de informações"
 L.Button_TestBars			= "Iniciar barras de teste"
 L.Button_MoveBars			= "Mover barras"
-L.Button_ResetInfoRange		= "Resetar quadros de informações/distância"
+L.Button_ResetInfoRange		= "Redefinir quadros de informações/distância"
```
Comment (Zidras) [inline DBM-GUI/localization.br.lua:81]: Same here with Resetar

File: DBM-GUI/localization.br.lua [hunk 7]
```diff
@@ -66,67 +66,123 @@ L.Area_ProfilesSetup		= "Guia de uso de perfis DBM"
...
-L.EnableModels				= "Habilitar modelos 3D nas opções de chefe"
+L.EnableModels				= "Ativar modelos 3D nas opções de chefe"
 L.ModelSoundOptions			= "Definir opção de som para o visualizador de modelos"
 L.ModelSoundShort			= "Curto"
 L.ModelSoundLong			= "Longo"
 
 L.ResizeOptions				= "Opções de redimensionamento"
 L.ResizeInfo				= "Você pode redimensionar a GUI clicando no canto inferior direito e arrastando."
-L.Button_ResetWindowSize	= "Resetar o tamanho da janela da GUI"
+L.Button_ResetWindowSize	= "Redefinir o tamanho da janela da GUI"
```
Comment (Zidras) [inline DBM-GUI/localization.br.lua:91]: =

File: DBM-GUI/localization.br.lua [hunk 8]
```diff
@@ -66,67 +66,123 @@ L.Area_ProfilesSetup		= "Guia de uso de perfis DBM"
...
---L.ShowSpellDescWhenExpanded	= "Continuar mostrando a descrição do feitiço quando os grupos forem expandidos"--Might not be used
-L.NoDescription				= "Esta habilidade não tem descrição"
-L.CustomOptions				= "Esta categoria contém opções personalizadas para uma habilidade ou evento que não possui ID de feitiço ou diário próprio. Essas opções foram agrupadas usando um ID manual personalizado para facilitar a criação de WeakAuras"
+L.UIGroupingOptions					= "Opções de agrupamento de IU (alterá-las requer recarregamento da IU)"
+L.GroupOptionsExcludeIcon			= "Excluir as opções de 'Ativar ícone' de serem agrupadas por feitiço (elas serão agrupadas na categoria categoria de 'Ícones')"
+L.GroupOptionsExcludePrivateAura 	= "Excluir as opções de som 'Aura Privada' de serem agrupadas por feitiço (elas serão agrupadas na categoria de 'Auras Privadas')"
+L.AutoExpandSpellGroups				= "Expandir automaticamente as opções agrupadas por feitiço"
+L.ShowWAKeys						= "Exibir as teclas WeakAuras ao lado dos nomes dos feitiços para ajudar a escrever WeakAuras usando os gatilhos do boss mod"
```
Comment (Zidras) [inline DBM-GUI/localization.br.lua]:
```suggestion
L.ShowWAKeys						= "Exibir as teclas WeakAuras ao lado dos nomes dos feitiços para ajudar a escrever WeakAuras usando os gatilhos do módulo de chefe"
```

File: DBM-GUI/localization.br.lua (hunk 5 above)
Comment (anon1231823) [reply DBM-GUI/localization.br.lua]: I just keep them in so people don't think they are missing in the future

Comment (Zidras) [reply DBM-GUI/localization.br.lua]: Even globals? that's seems wildly unnecessary. Why not comment the line out?

File: DBM-GUI/localization.br.lua (hunk 3 above)
Comment (anon1231823) [reply DBM-GUI/localization.br.lua:44]: Blizz uses Redefinir, although I agree both are usable I just tend to make everything follow blizz localization
image

Comment (Zidras) [reply DBM-GUI/localization.br.lua:44]: Agreed.

File: DBM-GUI/localization.br.lua (hunk 5 above)
Comment (anon1231823) [reply DBM-GUI/localization.br.lua]: Ok I commented the line out

**Review by MysticalOS — APPROVED** [review]

**Pull request merged**